server/webhook.go

```diff
@@ -329,3 +335,24 @@ func (p *Plugin) notify(ji Instance, parsed *parsedJIRAWebhook, text string) {
 		p.errorf("notify: %v", err)
 	}
 }
+
+func (p *Plugin) GetWebhookURL(teamId, channelId string) (string, error) {
+	cf := p.getConfig()
+
+	team, appErr := p.API.GetTeam(teamId)
+	if appErr != nil {
+		return "", appErr
+	}
+
+	channel, appErr := p.API.GetChannel(channelId)
+	if appErr != nil {
+		return "", appErr
+	}
+
+	v := url.Values{}
+	secret, _ := url.QueryUnescape(cf.Secret)
```

Comment:
Just for my own understanding, why are you unescaping the cf.Secret? Is there any reason it would be escaped before storing in the config?

Reply:
That's actually a good question. I might have gone too far there - never really found the code that generated the value. I'll double-check.